examples/TO_MIGRATE/visualization/animate_sedov_csv.py

```
@@ -164,7 +164,7 @@ def load_snapshot(filename):
 
         # Read header and data
         f.seek(0)
-        lines = [l for l in f.readlines() if not l.startswith("#")]
+        lines = [l for l in f if not l.startswith("#")]
```

Ruff (0.16.0)

[error] Ambiguous variable name: `l`

(E741)